packages/sources/instruxi-sxt/CHANGELOG.md

```diff
@@ -0,0 +1,99 @@
+# @chainlink/ion.au-adapter
```

Comment:
This is a new EA should only have one version

Reply:
Nothing should have been changed for ion.au, i only added instruxi-sxt

Reply:
Re-generate this file

Reply:
Which command will regenerate this

Reply:
Simply leave this file blank, we have github actions that will auto-populate this file

Reply:
This has been done on most recent push. Let me know what else needs to be done